index.html

```diff
@@ -9,22 +9,17 @@
 <body>
     <h2>Buffalo, NY</h2>
     <h3>Best Pizza in the Town!</h3>
-    <h4>Our Story
-       
-    </h4>
-    <img src="img/tmp.gif" alt="PizzaParty" width="300" height="400">" 
-    
-    
+    <img src="img/PizzaParty.jpg" alt="Two slices of Pizza holding hands" width="300" height="400">
+    <img src="img/tmp.gif" alt="DJ CAT" width="300" height="400">
 
     <ul>
-        <a href="./index.html">Best Pizza in Town!</a>
-        <li><a href="./ourstory.htm">Our Story</a></li>
-        <li><a href="./Menu.htm"> Menu </a></li>
-        <li><a href="./Reservations.htm">Reservations</a></li>
-        <li><a href="./contactus.htm">Connect with Us!</a></li>
+        <a href=" ./index.html">Best Pizza in Town!</a>
+        <li><a href="./ourstory.html">Our Story</a></li>
+        <li><a href="./Menu.html"> Menu </a></li>
+        <li><a href="./Reservations.html">Reservations</a></li>
+        <li><a href="./contactus.html">Connect with Us!</a></li>
```

Comment:
actually all of these paths are wrong. there is no need for the `./`

Reply:
hit "resolve conversation"  by accident

Reply:
this is fixed